uaclient/cli.py

```diff
@@ -69,6 +69,17 @@
 
 UA_COLLECT_LOGS_FILE = "ua_logs.tar.gz"
 
+NEW_VERSION_NOTICE = (
+    "\n"
+    + messages.BLUE_INFO
+    + """\
+ A new version is available: {version}
+Please run:
+    sudo apt-get install ubuntu-advantage-tools
+to get the latest version with new features and bug fixes.
+"""
```

Comment:
I don't think we need the extra newline
```suggestion
to get the latest version with new features and bug fixes."""
```